components/pickerbutton/stories/pickerbutton.stories.js

```diff
@@ -104,15 +86,6 @@ export default {
 			},
 			control: "boolean",
 		},
-		isKeyboardFocused: {
-			name: "Keyboard Focused",
-			type: { name: "boolean" },
-			table: {
-				type: { summary: "boolean" },
-				category: "State",
-			},
-			control: "boolean",
-		},
 		position: {
```

Comment:
Is there an expected visual change with position? I know you didn't add this but it does not seem to have an effect.

Reply:
It should have an affect on the Spectrum version. One side of the button does not have a border radius and that side should switch sides based on the position. Do you not see that?

![Screenshot 2023-06-23 at 10 14 51 AM](https://github.com/adobe/spectrum-css/assets/99203545/2505c038-04d3-4ac4-a554-6b4403debb2b)
![Screenshot 2023-06-23 at 10 14 56 AM](https://github.com/adobe/spectrum-css/assets/99203545/6969109e-e6d1-4c5c-9db8-9a7203e31a52)

Reply:
ahh ok yes I do see I think I might of been viewing it as express so was not seeing a difference.